activity_browser/layouts/tabs/activity.py

```diff
@@ -39,7 +46,7 @@ def open_activity_tab(self, key: tuple, read_only: bool = True) -> None:
         """Opens new tab or focuses on already open one."""
         if key not in self.tabs:
             act = bd.get_activity(key)
-            if not bc.is_technosphere_activity(act):
+            if act.get("type") not in bd.labels.node_types:
                 return
```

Comment:
Can we please give some feedback here instead of doing nothing? It is currently very confusing. For example, a popup saying that the current node can't be displayed because it isn't a process or a product.